patch/cisco-Enable-static-memory-reservation-for-OIRable-PCIe-de.patch

```diff
@@ -0,0 +1,198 @@
+commit 9a57f4339767fb8129ee54e9d1714af59d4a6937
+Author: balsup <balsup@contoso.com>
+Date:   Fri Aug 14 00:09:18 2020 +0000
+
+    Reserve address space for powered-off devices behind bridges
```

Comment:
is this upstreamed to linux?

Reply:
This is a Cisco ASIC specific change and is not upstreamed to kernel.org

Reply:
As replied in the private email communication, that it’s Cisco ASIC specific is not an argument against upstreaming.